src/sidebar/components/test/MarkdownEditor-test.js

```diff
@@ -61,6 +61,16 @@ describe('MarkdownEditor', () => {
     );
   }
 
+  function typeInTextarea(wrapper, text, key = undefined) {
+    const textarea = wrapper.find('textarea');
+    const textareaDOMNode = textarea.getDOMNode();
+
+    textareaDOMNode.value = text;
+    textareaDOMNode.selectionStart = text.length;
+
+    textarea.simulate('keyup', { key });
+  }
```

Comment:
Moved up here to make it accessible to other test contexts. Additionally replaced a native event dispatched inside `act(...)` with a call to `simulate(...)`.